README.md

```diff
@@ -5,105 +5,127 @@
 [![Trivy](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/actions/workflows/trivy-scan.yml)
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-ibm.ibmmq.svg)](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/commits/main)
 
-The `ballerinax/ibm.ibmmq` library provides an API to connect to an IBM MQ server using Ballerina.
+[IBM MQ](https://www.ibm.com/products/mq) is a powerful messaging middleware platform designed for facilitating reliable
+communication between disparate systems and applications. IBM MQ ensures the secure and orderly exchange of messages
+asynchronously, decoupling senders and receivers for efficient and scalable communication. It supports both
+point-to-point and publish/subscribe messaging models via queues and topics.
 
-This library is created with minimal deviation from the IBM MQ java client API to make it easy for the developers who are used to working with the IBM MQ java client.
+The `ballerinax/ibm.ibmmq` package provides an API to connect to an IBM MQ server using Ballerina.
```

Comment:
Is there any particular version/code name of the IBM MQ side we support? If yes, It'd be great to include that information :) ?

Reply:
Fixed with [3d2e8413695f1c4aded7605c26c918cc6314ba76](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/pull/37/commits/3d2e8413695f1c4aded7605c26c918cc6314ba76)